vassal-app/src/main/java/VASSAL/counters/PropertySheet.java

```diff
@@ -152,7 +164,7 @@ public void actionPerformed(ActionEvent event) {
 
   public PropertySheet() {
     // format is propertysheet;menu-name;keystroke;commitStyle;backgroundRed;backgroundGreen;backgroundBlue
-    this(ID + ";Properties;P;;;;", null);
+    this(ID + ";" + Resources.getString("Editor.PropertySheet.component_type") + ";P;;;;", null); //NON-NLS
```

Comment:
Make this key Editor.PropertySheet.default_command rather than Editor.PropertySheet.component_type